### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.5 - es. 3.10 (21 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#297] [Esercizi M-Z]
<img width="954" alt="immagine" src="https://github.com/user-attachments/assets/8435fad5-7399-4303-a4a8-01df5b33f850">

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/5db899ea-ade1-43cf-9922-07ec1dc44016)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/22805112-4abf-4e08-9c67-f30a426b302b)

**Comment by Andrii04** [comment]
![IMG_0719](https://github.com/user-attachments/assets/6b54ffd0-c0fd-4125-86eb-0e833461e360)

(Mi viene 0.21 invece di 0.22)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/23fc1ea8-f636-43bb-aec6-b921d7af0f4f)
